devenv/lib/repository.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import os.path
+from configparser import ConfigParser
+
+from devenv.lib.config import get_config
+from devenv.lib.fs import ensure_binroot
+
+
+class Repository:
+    path: str
+    name: str
+    config_path: str
```

Comment:
these annotations here don't do anything (and in general should be left off) -- mypy can infer the types from `__init__` and it's more likely these get out of sync (or break in mysterious ways since the declarations aren't checked for actual presence after `__init__`) https://www.notion.so/sentry/sentry-typing-guidelines-best-practices-471d5aacf262476188982ce70f909e00?pvs=4#4d926cb8ed3647f79f0e9164671705c1